src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build_task.py

```diff
@@ -138,6 +139,30 @@ def acr_build_task_show(cmd,
             raise
         logger.warning("Could not get build task details. Build task basic information is printed.")
 
+    if not with_secure_properties:
+        return build_task
+
+    try:
+        source_repository = client.list_source_repository_properties(resource_group_name,
+                                                                     registry_name,
+                                                                     build_task_name)
+        setattr(build_task, 'sourceRepository', source_repository)
+    except CloudError as e:
+        if e.status_code != 403:
+            raise
+        # pylint: disable=line-too-long
+        logger.warning("No permission to get source repository secret properties.")
```

Comment:
Update the use of `secret properties` to `secure properties`, be consistent with what the flag says.